offload/src/OpenMP/OMPT/Callback.cpp

```diff
@@ -27,14 +27,77 @@
 #undef DEBUG_PREFIX
 #define DEBUG_PREFIX "OMPT"
 
+using namespace llvm::omp::target::ompt;
+
 // Define OMPT callback functions (bound to actual callbacks later on)
 #define defineOmptCallback(Name, Type, Code)                                   \
   Name##_t llvm::omp::target::ompt::Name##_fn = nullptr;
 FOREACH_OMPT_NOEMI_EVENT(defineOmptCallback)
 FOREACH_OMPT_EMI_EVENT(defineOmptCallback)
 #undef defineOmptCallback
 
-using namespace llvm::omp::target::ompt;
+int ompt_get_device_num_procs(ompt_device_t *device) { return 0; }
+
+ompt_device_time_t ompt_get_device_time(ompt_device_t *device) { return 0; }
+
+double ompt_translate_time(ompt_device_t *device, ompt_device_time_t time) {
+  return 0;
+}
+
+ompt_set_result_t ompt_set_trace_ompt(ompt_device_t *device,
+                                      unsigned int enable, unsigned int etype) {
+  return ompt_set_error;
```

Comment:
I would not choose `ompt_set_error`, as tools could think they did something wrong (e.g. tried to use `target_emi` and `target`, which is not allowed). 
We should return `ompt_set_never` instead.

Reply:
`ompt_set_error` could still be used when trying to call this function outside of the `ompt_callback_device_initialize` callback, see the OpenMP 5.2 spec.

>  If the `ompt_set_trace_native` or the `ompt_set_trace_ompt` runtime entry point is called outside a device initializer, registration of supported callbacks may fail with a return code of `ompt_set_error`.